Update Line Chart Props (#943)
vtex/styleguide: react/components/EXPERIMENTAL_Charts/LineChart/README.md

```jsx noeditor
const sampleData = require('./multilineSample').default;
const keys = ['uv', 'pv', 'amt', 'gte', 'yaw'];
const customSchema = {container: { height: 200, width: '40%'}};
<LineChart
  data={sampleData}
  dataKeys={keys}
  xAxisKey='name'
  schema={customSchema}
/>
```

> - Avoid to do grid competes visually with the data.

Reviewer: Something wrong here in the sentence.

Author: the `to do` should be `makes`? :thinking:

```jsx noeditor
const sampleData = require('./multilineSample').default;
const keys = ['uv', 'pv', 'amt', 'gte', 'yaw'];
const customConfig = {container: { height: '40%', width: '40%'}};
<LineChart
  data={sampleData}
  dataKeys={keys}
  xAxisKey='name'
  config={customConfig}
/>
```

> - Avoid makes grid competes visually with the data.

Reviewer: ```suggestion
- Avoid the grid from visually competing with the data.
```

```

#### xAxis or yAxis

> This axis property are responsible to change visual appearence of the axis in the chart.

Reviewer: ```suggestion
This axis property is responsible to change the visual appearance of the axis in the chart.
```

- `tickLine`: If set true, axis tick lines will be drawn.
- `axisLine`: If set true, axis line will be drawn.

> - `hide`: If set true, the axis do not display in the chart..

Reviewer: ```suggestion
- `hide`: If set true, the axis does not display in the chart.
```

- `tick`: Renders a label of the axis. If you pass your component instance, it will receive
the props needed to render the label, like the `x` and `y` position of the tick in the line axis, the `payload` which represents the data
and the `fill` the label color. As you can see below:

```js
const CustomizedLabel = (props) => {
  const {
    x, y, fill, payload,
  } = props

  return (<text x={x} y={y} fill={fill} textAnchor="middle">{`${payload.value.toUpperCase()}`}</text>)
};

const sampleData = require('./sampleData').default;
const keys = ['customers'];

<LineChart
  data={sampleData}
  dataKeys={keys}
  xAxisKey='hour'
  schema={{
    container: {height: 300, width: '100%'}, 
    xAxis:{tick: <CustomizedLabel />}
  }}
/>
```

#### grid
The grid property is responsible to show a grid inside the chart.

> - `horizontal`: If set true, horizontal grid lines will be drawn.

Reviewer: ```suggestion
- `horizontal`: If set to `true`, horizontal grid lines will be drawn.
```

> - `vertical`: If set true, vertical grid lines will be drawn.

Reviewer: ```suggestion
- `vertical`: If set to `true`, vertical grid lines will be drawn.
```

```js
const sampleData = require('./sampleData').default;
const keys = ['customers'];

<LineChart
  data={sampleData}
  dataKeys={keys}
  xAxisKey='hour'
  schema={{ 
    xAxis:{tick: true},
    grid: {vertical: true, horizontal: true}
  }}
/>
```

#### container

> The container property is responsible to define the size of box that will render the chart.

Reviewer: ```suggestion
The container property is responsible to define the size of the box that will render the chart.
```

- `height`: The percentage value of the chart's width or a fixed width.
- `width`: The percentage value of the chart's width or a fixed height.

```js
const sampleData = require('./sampleData').default;
const keys = ['customers'];

<LineChart
  data={sampleData}
  dataKeys={keys}
  xAxisKey='hour'
  schema={{
    container: {height: 150, width: '80%'}, 
    xAxis: {tick: true}, 
    grid: {vertical: false, horizontal: false}
  }}
/>
```

### Line props

> - `type`: The interpolation type of line. For more details, check the [recharts types](http://recharts.org/en-US/api/Line#type)

Reviewer: We could maybe directly bring the most common ones here.

Author: I think in make something like [this](https://formidable.com/open-source/victory/gallery/interpolation)

Reviewer: You mean the select option?

Author: Yes!!

Reviewer: Sounds great!

Author: Done!! :smile:
